Library/Formula/swift.rb

```diff
@@ -77,20 +79,19 @@ def install
         "--build-args=-j#{ENV.make_jobs}",
         "--lldb-use-system-debugserver",
         "--install-prefix=#{prefix}",
-        "--darwin-deployment-version-osx=#{MacOS.version}",
```

Comment:
Linuxbrew attempts to keep compatibility with Mac. Please change this line to (unttested):
```ruby
("--darwin-deployment-version-osx=#{MacOS.version}" if OS.mac?),
```